samples/sample_bfbs.cpp

```diff
@@ -17,8 +17,9 @@
 #include "flatbuffers/idl.h"
 #include "flatbuffers/util.h"
 #include "monster_generated.h"  // Already includes "flatbuffers/flatbuffers.h".
+#include "monster_test_bfbs_generated.h" // Generated using --bfbs-comments --bfbs-builtins --cpp --bfbs-gen-embed
```

Comment:
This probably shouldn't be in the sample, since this is niche functionality. Samples are for the common case. Move this to the tests?

Reply:
I see, seem like a good place to give an example. I'll remove.

Reply:
I had this in a the test before in the old PR but there was a comment to remove it since it wasn't a common case. It would be nice if there was somewhere we can show a little how to use it.

Reply:
@aardappel Might it be better to make another sample?
For example, `sampel_embedded` which doesn't use file-io to operate with data?

Reply:
Good point, I'll add that and see what you guys think!

Reply:
Like I said, I think this is a very niche feature, it doesn't need its own sample. A mention in the docs or tests is sufficient.

Reply:
@aardappel Any suggestion where I would put this as a test? Don't want to create something new that doesn't fit good. How about creating a bfbs folder inside tests and call it test_bfbs_embed.cpp

Reply:
just test it directly from test.cpp?

Reply:
@aardappel I have move the sample stuff to a test instead. See what you think of this instead.